rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/synonyms/70_synonym_rule_delete.yml

```diff
@@ -0,0 +1,62 @@
+setup:
+  - skip:
+      version: " - 8.9.99"
+      reason: Introduced in 8.10.0
+  - do:
+      synonyms.put:
+        synonyms_set: test-synonyms
+        body:
+          synonyms_set:
+            - synonyms: "hello, hi"
+              id: "test-id-1"
+            - synonyms: "bye => goodbye"
+              id: "test-id-2"
+            - synonyms: "test => check"
+              id: "test-id-3"
+
+---
+"Delete synonym rule":
+  - do:
+      synonym_rule.delete:
+        synonyms_set: test-synonyms
+        synonym_rule: test-id-2
+
+  - match:
+      acknowledged: true
+  - match: { reload_analyzers_details._shards.total: 0 }
```

Comment:
Is there any yaml test we can do to ensure that reload analyzers returns a non-zero value?

Reply:
Yes, that is covered in the analyzers side, in [this test](https://github.com/elastic/elasticsearch/blob/75729363e2426a2dd108b2e57f1082bde21c288d/modules/analysis-common/src/yamlRestTest/resources/rest-api-spec/test/analysis-common/70_synonyms_from_index.yml#L1)